packages/coachmark/src/Coachmark.ts

```diff
@@ -13,32 +13,346 @@ governing permissions and limitations under the License.
+        return html`
+            <sp-button-group class="spectrum-ButtonGroup buttongroup">
+                ${when(
+                    this.secondaryCTA,
+                    () => html`
+                        <sp-button
+                            treatment="outline"
+                            variant="secondary"
+                            @click=${this.handleSecondaryCTA}
+                        >
+                            ${this.secondaryCTA}
+                        </sp-button>
+                    `
+                )}
+                ${when(
+                    this.primaryCTA,
+                    () => html`
+                        <sp-button
+                            size="s"
+                            treatment="outline"
+                            variant="primary"
+                            @click=${this.handlePrimaryCTA}
+                        >
+                            ${this.primaryCTA}
+                        </sp-button>
+                    `
+                )}
+            </sp-button-group>
+            <sp-button-group
+                class="spectrum-ButtonGroup buttongroup-mobile"
+                size="s"
+            >
+                ${when(
+                    this.secondaryCTA,
+                    () => html`
+                        <sp-button
+                            variant="secondary"
+                            treatment="outline"
+                            icon-only
+                            aria-label="previous"
+                            @click=${this.handleSecondaryCTA}
+                        >
+                            <sp-icon-chevron200
+                                size="s"
+                                class="spectrum-UIIcon-ChevronLeft75"
+                                slot="icon"
+                            ></sp-icon-chevron200>
+                        </sp-button>
+                    `
+                )}
+                ${when(
+                    this.primaryCTA,
+                    () => html`
+                        <sp-button
+                            size="s"
+                            treatment="outline"
+                            variant="primary"
+                            @click=${this.handlePrimaryCTA}
+                        >
+                            ${this.primaryCTA}
+                        </sp-button>
+                    `
+                )}
+            </sp-button-group>
+        `;
+    }
 
-    @property({ reflect: true })
-    public variant: 'dark' | 'light' | '' = '';
+    // render steps
+    protected renderSteps = (): TemplateResult => {
+        return html`
+            <div class="step" role="status">
+                <span aria-live="polite">
+                    ${this.currentStep} of ${this.totalSteps}
+                </span>
+            </div>
+        `;
+    };
+    // render action menu
+    protected renderActionMenu = (): TemplateResult => {
```

Comment:
```suggestion
    protected renderActionMenu(): TemplateResult => {
```

Same thing here. de-anonymise this function!

Reply:
We have to use an arrow function since this will become undefined here when we use `when` directive